browser/browsing_data/BUILD.gn

```diff
@@ -12,6 +12,7 @@ source_set("browsing_data") {
 
   deps = [
     "//base",
+    "//brave/common/extensions/api",
     "//chrome/common",
```

Comment:
I think you need to add a dep for `//extensions/browser` too - since `event_router.h` is included

Reply:
thanks - I also surrounded in a `if (enable_extensions) ...`